support/runtime-upgrade/index.js

```diff
@@ -0,0 +1,57 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+
+var argv = require('minimist')(process.argv.slice(2));
```

Comment:
Your example just shows providing arguments in sequence, doesn't minimist require arguments to be named, like `--node-address wss://...`?